internal/service/acm/certificate_data_source.go

```diff
@@ -45,6 +45,7 @@ func DataSourceCertificate() *schema.Resource {
 						acm.KeyAlgorithmEcSecp521r1,
 						acm.KeyAlgorithmRsa1024,
 						acm.KeyAlgorithmRsa2048,
+						acm.KeyAlgorithmRsa3072,
```

Comment:
Hi @Victorion! Rather than add the missing algorithm, we can actually use the `Values` collection so as new ones are added, the provider will automatically update whenever the `aws-go-sdk` is updated, this would look like:

```
ValidateFunc: validation.StringInSlice(KeyAlgorithm_Values(), false),
```

Reply:
thanks, @breathingdust !
I replaced this with `KeyAlgorithm_Values`